src/main/scala/uk/gov/homeoffice/cirium/services/feed/Cirium.scala

```diff
@@ -128,15 +143,15 @@ object Cirium {
 
     val latestItemActor: ActorRef = system.actorOf(Props(classOf[CiriumLastItemActor]), "latest-item-actor")
 
-    def start(step: Int): Future[Source[CiriumTrackableStatus, Cancellable]] = {
+    def start(step: Int)(implicit executionContext: ExecutionContext): Future[Source[CiriumTrackableStatus, Cancellable]] = {
```

Comment:
Could move these two to the class as well as we already have an implicit for the actor system